diff --git a/src/logger/index.ts b/src/logger/index.ts
index f006b151..cfff3d79 100644
--- a/src/logger/index.ts
+++ b/src/logger/index.ts
@@ -339,12 +339,14 @@ class Logger {
     logFlags.fatal = true
     logFlags.important_as_fatal = true
     logFlags.playback = false
+    logFlags.seqdiagram = true
   }
 
   setDisableAllFlags() {
     for (const [key, value] of Object.entries(logFlags)) {
       logFlags[key] = false
     }
+    logFlags.seqdiagram = true
   }
 
   setErrorFlags() {
@@ -357,6 +359,7 @@ class Logger {
     logFlags.important_as_error = true
 
     logFlags.playback = false
+    logFlags.seqdiagram = true
 
     //temp debug
     // logFlags.aalg = true
@@ -379,6 +382,8 @@ class Logger {
     logFlags.important_as_fatal = true
     logFlags.important_as_error = true
 
+    logFlags.seqdiagram = true
+
     //logFlags.rotation = true
   }
 
